mita/controllers/nodes.py

```diff
@@ -67,8 +67,8 @@ def idle(self):
                 jenkins_user = conf.jenkins['user']
                 jenkins_token = conf.jenkins['token']
                 conn = jenkins.Jenkins(jenkins_url, jenkins_user, jenkins_token)
-                # FIXME: We need to see if this jenkins_node exists first
```

Comment:
this still holds true though. We would need to check for `node_exists(self.node.jenkins_name)`

https://python-jenkins.readthedocs.org/en/latest/api.html#jenkins.Jenkins.node_exists